### ISSUE TITLE
Wizard: SelectOneMenu inside wizard tab shows the same dropdown panel twice onclick when I open the tab using javascript for more than one time

### ISSUE THREAD

**Issue by Nickath** [primefaces/primefaces#8799]
I have a **@ViewScoped** bean managing an .xhtml page containing **3 tab panels**, in the second panel I have some `<p:selectOneMenu` components. 

I am **navigating through the tabs using javascript**  for example: `"PF('wizard').loadStep(PF('wizard').cfg.steps[1], false);" `. When I navigate for the first time in the second tab, everything works fine, but **when I get there for the second or more time** and my first move is to click on any of selectOneMenus, the dropdown panel is **shown twice**, one in the dropdown menu position and one in the upper left corner of the page, 

**.xhtml page** Code sample below:

`
    
    <h:form id="wizardForm">
      <p:wizard id="wizard" showNavBar="false">

           <p:tab id="tab0" title="Tab 0">
                 <p:panel> Tab 0 </p:panel>
           </p:tab>

           <p:tab id="tab1" title="Tab 1">
                 <p:panel> 
                       <p:selectOneMenu  value="#{myBean.product">
                           <f:selectItems value="#{myBean.products}"/>
                          <p:ajax event="change" update="@this"/>
                       </p:selectOneMenu>
                  </p:panel>
           </p:tab>

            <p:tab id="tab2" title="Tab 2">
                 <p:panel> Tab 2 </p:panel>
           </p:tab>
      </p:wizard>
    </h:form>
           
   

`

   

### PrimeFaces edition

Community

### PrimeFaces version

11.0.0

### JSF version

2.2.20

### Browser(s)

Chrome, Firefox

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by Nickath** [comment]
> Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project. It is the only way developers can debug your problem to help.

I struggled to reproduce the bug in this environment but I did not manage to although I tried to transform the context to be as similar as possible to that of mine application. Nevertheless, I managed to solve my issue into my application by adding the ` appendTo="@this"` property into my `<p:selectOneMenu>`, so I guess the issue can be closed. 

Thanks for following up!
